src/Sulu/Bundle/WebsocketBundle/Resources/public/js/component/websocket-manager/client/ajax.js

```diff
@@ -36,7 +36,12 @@ define(['websocket/abstract', 'jquery'], function(Client, $) {
             data: {message: this.generateMessage(handler, message, {})}
         }).then(function(data) {
             this._onMessage.notify(data);
-            def.resolve(data.handler, data.message);
+
+            if (!data.error) {
```

Comment:
you get following informations:

```php
        return array(
            'handler' => $name,
            'message' => $message,
            'options' => $options,
            'error' => $error
        );
```

* `handler`: requested handler
* `message`: result of handler
* `options`: requested options
* `error`: flag which indicates if there was an error or not